source/js/package.json

```diff
@@ -1,14 +1,16 @@
 {
   "name": "astronomy-engine",
-  "version": "1.1.3",
+  "version": "2.0.0",
```

Comment:
before you `npm publish` this version, we need to setup some example code in the Angular website to see how it works

Reply:
OK. Let me know if you can use some help creating the example. Also, I'm not sure if you are ready for me to merge your changes yet. No hurry... please take your time; I just didn't want you to be waiting on me, when I didn't realize you were waiting.

I'm excited by what you are doing. I think this is going to make the JavaScript version much more useful to people. I can see now that having separate versions for the browser and node.js is a small price to pay in exchange for the much better type checking.

Reply:
Indeed, I liked this solution so far, if test are passing everything is working as it's supposed right?

I think we're ready to merge, unless you find a blocker :)

Reply:
OK, great! I will take a look at this tomorrow (Monday) morning.